tests/Unit/NumericalAlgorithms/Spectral/Test_Filtering.cpp

```diff
@@ -80,4 +80,55 @@ SPECTRE_TEST_CASE("Unit.Numerical.Spectral.ExponentialFilter",
     }
   }
 }
+
+template <Spectral::Basis BasisType, Spectral::Quadrature QuadratureType>
+void test_zero_lowest_modes() noexcept {
+  CAPTURE(BasisType);
+  CAPTURE(QuadratureType);
+  for (size_t num_pts =
+           Spectral::minimum_number_of_points<BasisType, QuadratureType>;
+       num_pts < Spectral::maximum_number_of_points<BasisType>; ++num_pts) {
+    CAPTURE(num_pts);
+    for (size_t number_of_modes_to_filter = 0;
+         number_of_modes_to_filter < num_pts; ++number_of_modes_to_filter) {
```

Comment:
Why not test the case where `number_of_modes_to_filter == num_pts`, which is allowed by the assertion in the filter function?